### ISSUE TITLE
Incorrect framerate in `resource/doc/ffmpeg.md`?

### ISSUE THREAD

**Issue by CrypticSignal** [Netflix/vmaf#1395]
In the [Using VMAF with FFmpeg](https://github.com/netflix/vmaf/blob/master/resource/doc/ffmpeg.md#using-vmaf-with-ffmpeg) section, we have the following example:
```
ffmpeg \
    -r 24 -i Seeking_30_480_1050.mp4 \
    -r 24 -i Seeking_10_288_375.mp4 \
    -lavfi "[0:v]setpts=PTS-STARTPTS[reference]; \
            [1:v]scale=720:480:flags=bicubic,setpts=PTS-STARTPTS[distorted]; \
            [distorted][reference]libvmaf=log_fmt=xml:log_path=/dev/stdout:model_path={your_vmaf_dir}/model/vmaf_v0.6.1.json:n_threads=4" \
    -f null -
```
We have `-r 24` in the command but the framerate of `Seeking_30_480_1050.mp4` and `Seeking_10_288_375.mp4` is 25, so shouldn't the framerate be set to 25 with `-r 25`? If so, I am happy to open a PR.

**Comment by nilfm99** [comment]
Hi, thanks for the heads up. I think we can change both to 25 no problem, and if you open a PR we'll welcome it.

For more detail on what's going on, that `-r` sets the input framerate, which tells ffmpeg how to interpret the timestamps in the input file. In this case it doesn't really matter what we set it to, as long as we set both to the same number - that will make the timestamps match and ffmpeg's framesync mechanism will feed both frames to libvmaf correctly. The VMAF results are the same whether you set both to `-r 24` or `-r 25`.
